src/subclass/list_box.rs

```diff
@@ -0,0 +1,208 @@
+
+use glib::translate::*;
+
+use glib::subclass::prelude::*;
+
+use super::container::ContainerImpl;
+use super::widget::WidgetImpl;
+use ListBoxClass;
+use WidgetClass;
+use ListBox;
+use ListBoxRow;
+
+pub trait ListBoxImpl: ListBoxImplExt + ContainerImpl + WidgetImpl {
+    fn row_activated(&self, list_box: &ListBox, row: &ListBoxRow) {
+        self.list_box_row_activated(list_box, row)
+    }
+
+    fn row_selected(&self, list_box: &ListBox, row: &ListBoxRow) {
+        self.list_box_row_selected(list_box, row)
+    }
+
+    fn activate_cursor_row(&self, list_box: &ListBox) {
+        self.list_box_activate_cursor_row(list_box)
+    }
+
+    fn select_all(&self, list_box: &ListBox) {
+        self.list_box_select_all(list_box)
+    }
+
+    fn selected_rows_changed(&self, list_box: &ListBox) {
+        self.list_box_selected_rows_changed(list_box)
+    }
+
+    fn toggle_cursor_row(&self, list_box: &ListBox) {
+        self.list_box_toggle_cursor_row(list_box)
+    }
+
+    fn unselect_all(&self, list_box: &ListBox) {
+        self.list_box_unselect_all(list_box)
+    }
```

Comment:
`move_cursor()` is missing here and also seems easy to add, then we have this one complete at least :)